src/pybind/mgr/dashboard/services/nvmeof_cli.py

```diff
@@ -45,3 +48,21 @@ def remove_nvmeof_gateway(_, name: str, daemon_name: str = ''):
         return 0, 'Success', ''
     except ManagedByOrchestratorException as ex:
         return -errno.EINVAL, '', str(ex)
+
+class NvmeofCLICommand(CLICommand):
+    def call(self,
+             mgr: Any,
+             cmd_dict: Dict[str, Any],
+             inbuf: Optional[str] = None) -> HandleCommandResult:
+        try:
+            ret = super().call(mgr, cmd_dict, inbuf)
+            format  = cmd_dict.get('format')
+            if format == 'json' or not format:
```

Comment:
If `not format` (default CLI output), from feedback from @caroav and @gbregman, I understand that (for backward compatibility) a tabular view should be provided, right?

Reply:
absolutely, but until we have more clarity regarding how we want to implement it, we decided to start with json and yaml only, when json is the default. I hope that makes sense :)